operator/roles/kiali-deploy/templates/dashboards/go.yaml

```diff
@@ -8,27 +8,53 @@ spec:
   discoverOn: "go_info"
   items:
   - chart:
-      name: "Threads"
+      name: "CPU ratio"
       spans: 6
-      metricName: "go_threads"
+      metricName: "process_cpu_seconds_total"
+      dataType: "rate"
+      aggregations:
+      - label: "pod_name"
+        displayName: "Pod"
+  - chart:
+      name: "RSS Memory"
+      unit: "bytes"
+      spans: 6
+      metricName: "process_resident_memory_bytes"
       dataType: "raw"
+      aggregations:
+      - label: "pod_name"
+        displayName: "Pod"
   - chart:
       name: "Goroutines"
       spans: 6
       metricName: "go_goroutines"
       dataType: "raw"
+      aggregations:
+      - label: "pod_name"
+        displayName: "Pod"
+  - chart:
+      name: "Heap allocation rate"
+      unit: "bytes/s"
```

Comment:
I will probably need to handle it like bitrate for auto-scaling : https://docker.pkg.github.com/kiali/k-charted/blob/master/web/react/src/utils/formatter.ts#L15